src/platform_impl/linux/wayland/seat/keyboard/handlers.rs

```diff
@@ -163,3 +155,575 @@ pub(super) fn handle_keyboard(
         }
     }
 }
+
+// ! ====================================================================================================== !
+// !                                         "INSPIRED" BY SCTK                                             !
+// ! ====================================================================================================== !
+
```

Comment:
It seems like this code doesn't really work. The keyboard repeat with it is entirely broken and doesn't work like it should. However do I really want to fix it here? I'm not sure, given that I have wayland-rs 0.30 branch and can figure out myself how to write it correctly.

However I think that the main issue with that impl is that you trigger key repeat for modifiers, and that you pause key repeat when pressing extra keys and such. So the Wayland impl in the current state is not really usable.

I won't go on the current impl, because this code will be destroyed anyway.

Reply:
Did you see the comment here https://github.com/rust-windowing/winit/commit/639e1d40b13a38d9545e452d9f7e7ae5b77e4740#r108100537?

In Neovide people have been using this fix for a while, and reported that it works fine https://github.com/neovide/winit/pull/12/files. So at least for us we would be happpy with that fix until you rewrite it.